lib/Dialect/TritonGPU/Transforms/Combine.cpp

```diff
@@ -561,11 +561,14 @@ class RematerializeForward : public mlir::RewritePattern {
 // -----------------------------------------------------------------------------
 //
 // -----------------------------------------------------------------------------
+namespace {
 static int computeCapabilityToMMAVersion(int computeCapability) {
   if (computeCapability < 80) {
     return 1;
   } else if (computeCapability < 90) {
     return 2;
+  } else if (computeCapability < 100) {
+    return 3;
   } else {
     assert(false && "computeCapability > 90 not supported");
```

Comment:
The error message doesn't seem consistent with `computeCapability < 100`. For instance, we don't support `computeCapability=95` right?

Reply:
Sorry, forget delete this line.